tools/test/audio/std_utils/fr_test_measure.m

```diff
@@ -33,15 +33,17 @@
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 
 %% Check if upper and lower mask is defined
-if length(test.fr_mask_flo) || length(test.fr_mask_fhi)
+if ~isempty(test.fr_mask_flo) || ~isempty(test.fr_mask_fhi)
```

Comment:
This to first cleanup patch if this is not fixing anything, does it? Personally I find length() easier to understand than "not is empty?"